mobile/src/components/Report.js

```diff
@@ -62,73 +63,50 @@ export default class Report extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      reportId: '',
-      reportName: '',
-      templateParameters: [],
-      currentPage: 1,
-      totalPages: 1,
-      pageSize: null,
-      maxPageSize: 2147483647,
       staticParams: ['pageSize', 'offset', 'orderBy'],
       tableHeaders: [],
       columnWidths: [],
-      tableData: [],
+      tableRows: [],
+      reportJson: null,
+      pageSize: 10,
+      currentPage: 1,
+      totalPages: 1,
+      totalValues: 0,
     };
 
-    this.fetchReport = this.fetchReport.bind(this);
     this.fetchNextPage = this.fetchNextPage.bind(this);
     this.fetchPrevPage = this.fetchPrevPage.bind(this);
     this.fetchFirstPage = this.fetchFirstPage.bind(this);
     this.fetchLastPage = this.fetchLastPage.bind(this);
     this.fetchPdf = this.fetchPdf.bind(this);
     this.fetchXls = this.fetchXls.bind(this);
+    this.setTableData = this.setTableData.bind(this);
```

Comment:
You do not need to bind function if you are using arrow function (btw. i think we could stick to one approach of writing funcions)

Reply:
done